uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -6,7 +6,7 @@
     </button>
     <h4 class="modal-title" ng-switch="isRole('admin')">
       <span ng-switch-when="true">New Document</span>
-      <span ng-switch-when="false">{{getDocumentType(document.activity_type_id)}}</span>
+      <span ng-switch-when="false">{{getDocumentType(document.activity_type_id) || 'Select Document Type'}}</span>
```

Comment:
For now, in  the title will remain "New Document" in edit mode. Not sure if we need to change to a different title for edit mode. But yes, doesn't fit for edit mode.